docs/learn/use-netlify-cms/use-netlify-cms.md

```diff
@@ -0,0 +1,202 @@
+---
+title: Use Netlify CMS with Your Scully Site
+lang: en
+position: 20
+---
+
+## [Overview](#overview)
+
+Scully is great for building fast, static websites. If you have a technical background, adding content to the site is relatively easy. You can edit the HTML directly, or add new markdown files for new blog posts. But less technical users benefit from having a content management system to edit the content. Netlify provides a CMS option that you can use, and getting it set up with Scully is pretty straightforward. In this post, you'll learn how to create a Scully site and enable Netlify CMS.
+
+## [First Steps](#first-steps)
+
+First up, create a new Angular application, and initialize Scully and the blog:
+
+```bash
+$ ng new scully-netlify-cms --routing
+...
+$ ng add @scullyio/init
+...
+$ ng g @scullyio/init:blog
+...
+```
+
+I'm not going to go over each of these steps in detail; that's best left for another post. Just know that those three commands create a new Angular application, initialize Scully, and add blog support to Scully. Next up, we need to create a couple modules and components to use for routing. You can do that like this:
```

Comment:
This is a docs page, not a post.

Reply:
Yes, I missed this in my review. I'll change that.